api/HTMLAreaElement.json

```diff
@@ -351,7 +351,7 @@
           "spec_url": "https://html.spec.whatwg.org/multipage/links.html#dom-hyperlink-origin-dev",
           "support": {
             "chrome": {
-              "version_added": "8"
+              "version_added": "32"
```

Comment:
Manually confirmed with https://mdn-bcd-collector.appspot.com/tests/api/HTMLAreaElement/origin and the test is so simple there's little room for doubt.

But if the Chrome data was wrong, so was Safari most likely. Here's the commit I think added this in Safari:
https://github.com/WebKit/WebKit/commit/fe3dc0b4f4d5406eeb5e77e962570bd8a9921cad
https://github.com/WebKit/WebKit/blob/fe3dc0b4f4d5406eeb5e77e962570bd8a9921cad/Source/WebCore/Configurations/Version.xcconfig

Can you test Safari too?